exchangelib/items.py

```diff
@@ -1317,5 +1317,38 @@ def __hash__(self):
         return super(Persona, self).__hash__()
 
 
+class ReadFlagChange(Item):
+    """
+    MSDN: https://msdn.microsoft.com/en-us/library/office/aa565609(v=exchg.150).aspx
+    """
+    ELEMENT_NAME = 'ReadFlagChange'
+    FIELDS = Item.FIELDS
+
+
+class CreateChange(Item):
+    """
+    MSDN: https://msdn.microsoft.com/en-us/library/office/aa565609(v=exchg.150).aspx
+    """
+    ELEMENT_NAME = 'Create'
+    FIELDS = Item.FIELDS + [
+        EWSElementField('changed_item', field_uri='Message', value_cls=Message)
```

Comment:
FYI I believe you need to add all the fields here (https://docs.microsoft.com/en-us/exchange/client-developer/web-service-reference/create-itemsync) to handle other element types.  I didn't check the other parser classes, but they also look a little think.

Thanks for the patch!